Yoast/javascript
# Fix stemming of Indonesian words beginning with m- with prefix -pem/mem

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Checks the exception list of Indonesian words beginning with _m_ if prefix -mem/pem is found, and stems only _-me/pe_ if the word is on the exception list.
* [Yoast SEO Premium] Improves word form recognition of Indonesian words beginning with -m after they get the -mem/pem prefix.


## Test instructions

This PR can be tested by following these steps:

* Make sure all current specs pass
* Add more examples to the specs and make sure they pass. To create the examples, take entries from the mBeginning list in the Indonesian morphology data file and add prefix mem-/pem- in front of them. Also try combining the prefix with relevant suffixes (mem- can occur with the -kan/-i suffixes, and pem- with the -an suffix).
* In Indonesian Premium: Add words beginning with m- to the keyphrase and the text/title/meta description. Make sure to include forms with _me_-/_pe_ in front of the word, also combined with possible suffixes (see previous point). See whether the forms are recognised.

## Quality assurance


Fixes https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-389
